src/test/scala/org/broadinstitute/dsde/workbench/sam/TestSupport.scala

```diff
@@ -49,6 +49,8 @@ import scala.concurrent.{Await, Awaitable, ExecutionContext}
   */
 trait TestSupport{
   def runAndWait[T](f: Awaitable[T]): T = Await.result(f, Duration.Inf)
+  def runAndWait[T](f: IO[T]): T = f.unsafeRunSync()
```

Comment:
nice!  we should make sure we remember to use this when writing tests in the future

Reply:
well... this was more so I didn't have to change a bunch of code (and intellij doesn't support inlining this)